src/deploy/nsfs.service

```diff
@@ -1,12 +1,15 @@
 [Unit]
 Description=The NooBaa nsfs service.
+After=gpfs-wait-mount.service
 
 [Service]
 Restart=always
 RestartSec=2
 User=root
 Group=root
-Environment=
+Environment=ENDPOINT_FORKS=0
```

Comment:
Should we also allow another place to configure this, like `system.json` (eg. `"forks": 8`) so that consumers of this service don't have to edit the unit file to configure the forks?

Reply:
I think that we can specify an .env file instead under the config dir, WDYT?

Reply:
On a second thought, I think `config-local.js` is good enough for that purpose.

Reply:
> I think that we can specify a .env file instead under the config dir, WDYT?
Makes sense!